packages/inventory/package.json

```diff
@@ -17,25 +17,20 @@
   "author": "Medusa",
   "license": "MIT",
   "devDependencies": {
-    "@medusajs/medusa": "*",
     "cross-env": "^5.2.1",
     "jest": "^25.5.4",
     "ts-jest": "^25.5.1",
     "typescript": "^4.4.4"
   },
   "dependencies": {
-    "@medusajs/modules-sdk": "*",
-    "awilix": "^8.0.0",
-    "typeorm": "^0.3.11"
+    "@medusajs/medusa": "workspace:^",
```

Comment:
question: Is that temporary? When the package will be deployed, not sure it works? Also, I suppose this deps is temporary until we extract the necessary elements right?

Reply:
+1 -- also, I think we should avoid core dependency in all modules.

Reply:
That is temporary. Reorganizing shared functions to remove the dependency of modules on the core is a bit [bigger](https://github.com/medusajs/medusa/pull/3393) task.